Description: aic8800_fdrv (USB): log the firmware patch table and overheard 4-address frames at debug level
Author: NomDeTom
Forwarded: no
Last-Update: 2026-10-11
---
Index: aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800d80n.c
===================================================================
--- aic8800.orig/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800d80n.c
+++ aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800d80n.c
@@ -180,7 +180,7 @@
             if ((dst[i] == 0x0) || (dst[i] == 0xFFFFFFFF)) {
                 break; // end of tbl
             }
-            AICWFDBG(LOGERROR, "patch_tbl:  %x  %x\n", dst[i], dst[i+1]);
+            AICWFDBG(LOGDEBUG, "patch_tbl:  %x  %x\n", dst[i], dst[i+1]);
             err = rwnx_send_dbg_mem_write_req(rwnx_hw, dst[i], dst[i+1]);
         }
         if (err) {
Index: aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800dc.c
===================================================================
--- aic8800.orig/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800dc.c
+++ aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_compat_8800dc.c
@@ -2749,7 +2749,7 @@
 
     if (!err && (i < size)) {
         for (i =(128/4); i < (size/4); i +=2) {
-            AICWFDBG(LOGERROR, "patch_tbl:  %x  %x\n", dst[i], dst[i+1]);
+            AICWFDBG(LOGDEBUG, "patch_tbl:  %x  %x\n", dst[i], dst[i+1]);
             err = rwnx_send_dbg_mem_write_req(rwnx_hw, dst[i], dst[i+1]);
         }
         if (err) {
Index: aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_rx.c
===================================================================
--- aic8800.orig/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_rx.c
+++ aic8800/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_rx.c
@@ -2401,8 +2401,8 @@
 
 #ifndef CONFIG_SUPPORT_4ADDR
 		if (hw_rxhdr->flags_is_4addr) {
-					dev_err(rwnx_hw->dev, "4addr Frame received (%d), skb->len:%u\n", hw_rxhdr->flags_vif_idx, skb->len);
-					print_hex_dump(KERN_ERR,"4addr ",DUMP_PREFIX_NONE, 16, 1, skb->data, skb->len, false);
+					dev_dbg(rwnx_hw->dev, "4addr Frame received (%d), skb->len:%u\n", hw_rxhdr->flags_vif_idx, skb->len);
+					print_hex_dump_debug("4addr ", DUMP_PREFIX_NONE, 16, 1, skb->data, skb->len, false);
 					dev_kfree_skb(skb);
 					goto end;
 		}
